modules/virt-editing-vm-web.adoc

```diff
@@ -30,28 +30,34 @@ endif::[]
 
 = Editing a {object} in the web console
 
-Edit select values of a {object} in the web console by clicking the pencil icon next to the relevant field. Other values can be edited using the CLI.
+You can edit a {object} by using the {product-title} web console or the command line interface.
 
-You can edit labels and annotations for any templates, including those provided by Red Hat. Other fields are editable for user-customized templates only.
+ifdef::virt-vm-template[]
+Editing a {object} does not affect virtual machines already created from that template.
+endif::virt-vm-template[]
 
 .Procedure
 
-. Click *Virtualization* -> *{object-gui}* from the side menu.
-
-. Optional: Use the *Filter* drop-down menu to sort the list of {object}s by attributes such as status, template, node, or operating system (OS).
+. Navigate to *Virtualization* -> *{object-gui}* in the web console.
 
+ifdef::virt-vm[]
 . Select a {object} to open the *{object-vm-details}* page.
 
 . Click any field that has the pencil icon, which indicates that the field is editable. For example, click the current *Boot mode* setting, such as BIOS or UEFI, to open the *Boot mode* window and select an option from the list.
+endif::virt-vm[]
+ifdef::virt-vm-template[]
+. Click the {kebab} Options menu beside a {object} and select *Edit*, *Edit boot source*, *Edit boot source reference*, *Edit labels*, or *Edit annotations*.
```

Comment:
Nit: this is just describing the UI which isn't helpful generally and is out of date as soon as a new item is added to the UI. What about the following:

```suggestion
. Click the {kebab} Options menu beside a {object} and edit the required item.
```

Reply:
We plan to update this assembly before each release because we can't keep up with the UI in the documentation. This assembly will be the 'source of truth' and will replace the plethora of modules that we cannot keep up to date. :-)